samples/msgext-ai-sentiment-analysis/nodejs/.env

```diff
@@ -0,0 +1,6 @@
+MicrosoftAppId=<<MicrosoftAppId>>
```

Comment:
Did you mention in Readme file to update these values manually?

Reply:
Yes, I mentioned.